### PR TITLE
Update tests to match lecture notes

### PR DESCRIPTION
**Pull request by Josehower**
The content committed during my Authentication and Authorization lectures did not match the lecture notes, breaking the tests.

In the meantime, I have temporarily worked around this so that CI is not failing on `main`:

- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/f6ddf1facc92efd26b82a42c64b43cb151a0bda3
- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/1b420063d2f1dcf2c1ec62efd88f868cc161ae27
- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/38dc1858298c5ee5942f4345dea6ac7724b9b572

But the correct solution will be to not diverge from the lecture notes in the Authentication and Authorization lectures - we will make sure to follow the notes next time.

Do not merge this PR yet - it should be merged after the cohort ends on July 18, 2023, so that the students do not see an inconsistent repo + lecture combination.

### CODE REVIEW

File: playwright/navigation.spec.ts [hunk 1]
```diff
@@ -45,11 +45,6 @@ test('navigation test', async ({ page }) => {
   await expect(page.getByAltText('cat sleeping').first()).toBeVisible();
 
   await page.getByRole('link', { name: 'animals' }).click();
-  await expect(page).toHaveURL('http://localhost:3000/login?returnTo=/animals');
```
Comment (karlhorky) [inline playwright/navigation.spec.ts:48]: I think this `expect(page).toHaveURL(...` shouldn't be removed completely, based on the temporary workaround commits here:

- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/f6ddf1facc92efd26b82a42c64b43cb151a0bda3
- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/38dc1858298c5ee5942f4345dea6ac7724b9b572

Comment (Josehower) [reply playwright/navigation.spec.ts:48]: 34b3d22

Comment (Josehower) [reply playwright/navigation.spec.ts:48]: 3cce018

Comment (Josehower) [reply playwright/navigation.spec.ts:48]: 7876d34

**Review by karlhorky — APPROVED** [review]

**Pull request merged**